packages/client/src/generation/TSClient/PrismaClient.ts

```diff
@@ -303,7 +346,9 @@ export class PrismaClientClass implements Generatable {
     protected readonly generator?: GeneratorConfig,
     protected readonly sqliteDatasourceOverrides?: DatasourceOverwrite[],
     protected readonly cwd?: string,
-  ) {}
+  ) {
+    this.clientExtensionsDefinitions = clientExtensionsDefinitions.bind(this)()
```

Comment:
The extension spans across the `Prisma` and default namespace, so we instantiate it right at the beginning.